ClientTests/TestHistory.swift

```diff
@@ -199,7 +199,7 @@ class TestHistory: ProfileTest {
             cb()
             return
         } else {
-            runRandom(&history, cmdIn: -1) { [history]_ in
```

Comment:
how was that working?

Reply:
what is `[history]_` going to do?

Reply:
I've seen `_` as an empty argument for a closure, but shouldn't there have been a comma? i.e. `[history], _`?

Reply:
there usually isnt a comma